prow/plugins/cla/cla_test.go

```diff
@@ -348,11 +425,20 @@ func TestCheckCLA(t *testing.T) {
 				Number:     3,
 				IssueState: tc.issueState,
 			}
+			cc := plugins.CLAConfig{
+				CLAContextNames: []string{"cla/linuxfoundation", "EasyCLA"},
+			}
+			var statuses []github.Status
+			for context, state := range tc.contexts {
+				statuses = append(statuses, github.Status{
```

Comment:
re: my "are the statuses sorted" comment above, your unit tests are relying on _this_ being the sort

what I lack is proof that you've verified this sort is what you actually see in production (show me code that presorts the statuses, or docs from github guaranteeing the order in which they deliver statuses)

Reply:
@spiffxp 

I hear you. But I could not find proof for sort order. Here is the docs for [getting combined status](https://docs.github.com/en/rest/reference/repos#get-the-combined-status-for-a-specific-reference)
and the [client code to get the same](https://github.com/google/go-github/blob/ac48d59eb13be558759e9dd1e6f8f75126a83a4e/github/repos_statuses.go#L114)

Reply:
maybe this is a sign you should sort them before iterating over them?  Or, flip the loop so you iterate over the context names that you care about in the order you care about, and then see if they're contained in combined statuses